### PR TITLE
rich the information of package.json

### PR DESCRIPTION
**Pull request by pvnr0082t**
add some information for package.json
- description
- author
- repository
- keywords
- homepage
- license

### CODE REVIEW

**Comment by agallou** [comment]
Thanks for the PR. 
Does this mean you need the package to be published on npm ?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,22 @@
 {
   "name": "highchartTable",
   "version": "1.1.0",
+  "description": "This jQuery plugin provides a simple way to convert HTML data tables to Highcharts graphs.",
+  "author": "agallou",
```
Comment (agallou) [inline package.json]: there is multiple contributors to the project. Could you change the author to "highchartTable Development Team (https://github.com/highchartTable/jquery-highchartTable-plugin/graphs/contributors)" ?

Comment (pvnr0082t) [inline package.json]: Of course. Author information updated. :+1:

**Comment by pvnr0082t** [comment]
> Does this mean you need the package to be published on npm ?

It's up to you:smiley: . These information are necessary in [cdnjs](https://github.com/cdnjs/cdnjs), so I just want to contribute to the upstream.

**Comment by agallou** [comment]
thanks @pvnr0082t

**Comment by pvnr0082t** [comment]
@agallou No problem. :facepunch:

**Pull request merged**